src/switchroot/ostree-remount.c

```diff
@@ -95,8 +95,11 @@ main(int argc, char *argv[])
       exit (EXIT_SUCCESS);
     }
 
-  do_remount ("/sysroot");
   do_remount ("/var");
 
+  struct stat stbuf;
+  if (lstat ("/sysroot/ostree/.sysroot-ro", &stbuf) < 0)
```

Comment:
If someone has a better idea than this magic file...I'm all ears.  We could do a kernel argument I guess, but it's not obviously better?